corehq/warehouse/models/staging.py

```diff
@@ -70,14 +76,23 @@ def field_mapping(cls):
         ]
```

Comment:
not part of this PR but curious why you renamed `last_modified` to `group_last_modified`. Same for others below.